### ISSUE TITLE
FTP Suspend Bug?

### ISSUE THREAD

**Issue by Dexxa1024** [sentora/sentora-core#131]
When you suspend an account, the web page and client panel suspends, But still allows you access to the ftp using the normal client accounts details,

so customers still have access to their ftp account even when the service is suspended

**Comment by MBlagui** [comment]
Ok we will check that.

**Comment by 5050** [comment]
Is't it correct ?
It enables a user to backup his account content when it is suspended, ie before a complete removal of his account.
I have the feeling that also disabling the ftp may be an option but must not be automatic.

**Comment by bobsta63** [comment]
Yeah i agree! - if you want to stop your users access altogether they should 'delete' instead of 'suspend'

**Comment by MBlagui** [comment]
I don't agree, when an admin suspend user account, it means we ban him access FTP/EMAIL/WEB.

In hosting this is not over being good and offering a way to backup. If the admin would allow that he would enable back the account and delete the websites or suspend them ( using vhost settings).

Some time we suspend an account to force a customer to pay his bills and if he is able to access his data some will grab them and flee away.

**Comment by bobsta63** [comment]
So just to confirm then, you're not talking about when the account is
suspended for bandwidth over usage? - Which is what I thought you meant?

**Comment by PhilipMcGaw** [comment]
I agree that suspend should suspend all services. 

If a user wants a backup. They can request a zip file from the admin.  

FTP doesn't give access to emails or other non web root assets. 

Philip McGaw
www.philipmcgaw.com
07969502077

@Philip_McGaw

**Comment by Dexxa1024** [comment]
Some customers and use hosting services to host XMBC media content for their media centres and can ftp stream them, if the ftp still works after suspend then they still get a service when they are suspended. or if a user uploads a script that's not legal and you need proof that they have used the account for illegal activities and you suspend them, They can still delete the files and then if you report it and they've deleted it.
Mostly every panel suspends every service when an accounts been suspended, it would be great if this panel can do the same.

**Comment by Dexxa1024** [comment]
@bobsta63  It maybe a good idea, lets say if the user doesnt pay on time, you delete the account and then he pays, Then you'd have to go through the hassle of resetting up his/hers account and everything working again, but then theres the other way where you just suspend and then unsuspend and the accounts back online in full contact.

I suspend clients and then when they dont pay within a vaild time frame then it gets deleted, if we just delete the account instead of suspending then that makes the host look bad, we just know people will be going on forums and complaining "this host didnt suspend my account but deleted it" lines :)

**Comment by bobsta63** [comment]
I see your point(s) - we'll work to implement in the next release!
